### DISCUSSION TITLE
How complete is the medrecon?

### DISCUSSION THREAD

**Discussion by JOL6754** [MIT-LCP/mimic-code#1437] [MIMIC-IV-ED]
Hi, I'm a medical doctor from Australia. In my hospital it is relatively common practice to 'freetext'  medication taken prior to presentation to hospital into the medical notes, and not input each individual med into the electronic record. It is a cumbersome process. Some free text, some input into the electronic record. Do you know if it is mandatory to input each medication into the record with this database?

I am considering a study investigating pre-admission exposure to various medications, however if meds are 'freetext' into the medical notes I strongly suspect this would not make it into the medrecon. 

Which may be a source of confounding. Ie sicker patients who need acute resuscitation may not have their medrecon complete, giving the illusion of no pre-hospital meds.

Thanks

**Comment by alistairewj** [comment]
Not sure I can speak to completeness, but for hospitalized patients they fill in the admission medications in the discharge summary (free-text).
